redbeams/src/test/java/com/sequenceiq/redbeams/service/dbconfig/DatabaseConfigServiceTest.java

```diff
@@ -56,6 +60,8 @@
 
     private static final String ENVIRONMENT_CRN = "environmentCrn";
```

Comment:
Not really a fix -- should we be doing environment CRN rather than Id? I've seen environment CRN elsewhere. This goes to @jadair-cloudera and @bhavanki as well.

Reply:
Either CRN or name can be used to identify an environment. Here, it's just a test value, so it appears fine to me.

Reply:
I don't mean here specifically, I mean in general -- FreeIPA uses environment CRN, for instance.